src/screens/OtherProfile/index.js

```diff
@@ -603,7 +528,8 @@ const OtherProfile = () => {
   };
 
   const handleDataFeed = () => {
-    const isFeedsShown = !blockStatus.blocked && !blockStatus.blocker;
+    if (isProfileFetching) return [];
+    const isFeedsShown = !(isBlocking || isBlocked);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isProfileFetching` check before returning feeds is a good practice as it ensures that the data is not accessed before it is fetched.



<!-- This is an auto-generated comment by OSS CodeRabbit -->